packages/grafana-ui/src/components/Text/Text.tsx

```diff
@@ -18,7 +18,7 @@ import { Tooltip } from '../Tooltip/Tooltip';
 
 import { customWeight, customColor, customVariant } from './utils';
 
-export interface TextProps {
+export interface TextProps extends Omit<React.HTMLAttributes<HTMLElement>, 'className'> {
```

Comment:
Do we want to omit `style` or is that already handled?

Reply:
yes!